docs/terminal/Tassie TCU/operations.md

```diff
@@ -4,3 +4,27 @@
 
 --8<-- "includes/abbreviations.md"
 
+## Positions
+
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | ---------------------|
+| Hobart Approach  | HBA | Hobart Approach  | 125.550       | HB_APP    |
+| Launceston Approach  | LTA | Launy Approach   | 123.800        | LT_APP                 |
+
+!!! Note
+    Hobart Approach may extend to Launy Approach and vice versa, callsigns remain the same.
+
+## Airspace
+The TAS TCU has an upper limit of FL245 within the following area:
+
+<figure markdown>
+![TAS TCU](img/TASTCU.png){ width="700" }
+</figure>
+
+## Hobart
+
+All aircraft should be kept on SIDs and STARs. If due to operational requirements, an aircraft is unable to accept the SID or STAR, Coordination with HUO may be required.
+
+## Launceston
+
+Visual approaches are preferred into Launceston, if due to operational requirements, an aircraft is unable to accept a visual approach, Coordination with **LT ADC** may be required.
```

Comment:
Launy has no STARs, however aircraft are now often instructed to track for 32L via IRSOM, NODAS, MLTSC which keeps the aircraft within CTA and away from the departures stream. 32L is usually the favourable runway. I expect launcy to have STARs at some point in the near future.